cpp/inc/bond/core/nullable.h

```diff
@@ -435,7 +427,7 @@ class nullable<T, Allocator, false>
     {
         if (_value)
         {
-            destroy(_alloc);
+            delete_value();
             _value = 0;
```

Comment:
Does this assignment to zero still make sense if, say, custom smart pointer types are being employed by the allocator?